packages/roosterjs-editor-types/lib/interface/VCell.ts

```diff
@@ -16,4 +16,7 @@ export default interface VCell {
      * Whether this cell is spanned from above
      */
     spanAbove?: boolean;
+
+    width?: number /* the width in pixel of the actual td (including border and padding) */;
```

Comment:
Please put the comment before the properties so that typedoc can generate document for these attributes, like what we have for other properties .